doc/porting/board_porting.rst

```diff
@@ -120,3 +120,30 @@ Board
 A board implements an SoC with all its features, together with peripherals
 available on the board that differentiates the board with additional interfaces
 and features not available in the SoC.
+
+Default board configuration
+***************************
+
+When porting Zephyr to a board, you must provide the board's default
+Kconfig configuration, which is used in application builds unless explicitly
+overridden.
+
+In order to provide consistency across the various boards and ease the work of
+users providing applications that are not board specific, the following
+guidelines should be followed when porting a board:
```

Comment:
Looks great! The only thing I would add to the end of this sentence is "when possible". Sometimes it is not possible to follow the guidelines, such as if the board does not have a network interface.